README.md

```diff
@@ -35,9 +36,75 @@ npm install -g @unstoppabledomains/resolution
 
 It should install binary named resolution in the default folder for your package manager. You can check it by running command `resolution -V` in command line. If everything is fine you will see the version installed.
 
-# Usage
+## Usage
+
+Create a new project.
+
+```shell
+mkdir test-out-resolution && cd $_
+yarn init -y
+yarn add @unstoppabledomains/resolution
+```
+
+Make a file, `script.js`.
+
+```javascript
+const {default: Resolution} = require('@unstoppabledomains/resolution')
```

Comment:
It doesn't work like that out of the box unfortunately, since our lib represents a CommonJS module, which doesn't support having both default and named exports. A user would have to do it like this then:
```
import resolution from '@unstoppabledomains/resolution'
const Resolution = resolution.default; // or resolution.Resolution
```
(importing it as default and then getting `default` property of the module object.. yeah)

Importing with
```
import { Resolution } from '@unstoppabledomains/resolution'
```
won't work either, because NodeJS for some reason doesn't support named imports from CommonJS

It's better to leave it untouched, imo

Reply:
How does it work with `import` within the website then?

Reply:
TypeScript